app/View/Users/notification_settings.ctp

```diff
@@ -0,0 +1,87 @@
+<?php
```

Comment:
What's with the sad field widths? :(

Reply:
![image](https://user-images.githubusercontent.com/3668672/189347104-30f23429-baae-40db-a672-29fe71b89b01.png)